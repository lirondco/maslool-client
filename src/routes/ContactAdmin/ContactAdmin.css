.ContactAdmin {
    margin: 2%;
    padding: 2%;
}

.ContactAdmin h4 {
    margin-top: 2%;
    margin-left: 2%;
    margin-right: 2%;
}

.ContactAdmin h2 {
    margin-bottom: 1%;
}

hr {
    border: 1px solid;
}

