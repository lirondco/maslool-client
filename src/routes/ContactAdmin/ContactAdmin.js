import React, { Component } from 'react'
import ContactForm from '../../components/ContactForm/ContactForm'
import './ContactAdmin.css'

export default class ContactAdmin extends Component {
    render() {
        return (
            <section className='ContactAdmin'>
                <h2>Contact Admins</h2>
                <hr />
                <h4>Send us your suggestions, comments, or general inquiries! If you want to suggest a new trail to be included, please include as much details as you can including a name, address, and a website.</h4>
                <ContactForm />
            </section>
        )
    }
}