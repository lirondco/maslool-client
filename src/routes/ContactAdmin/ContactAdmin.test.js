import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ContactAdmin from './ContactAdmin'

describe(`ContactAdmin route`, () => {
  it('renders the complete page', () => {
    const wrapper = shallow(<ContactAdmin />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})