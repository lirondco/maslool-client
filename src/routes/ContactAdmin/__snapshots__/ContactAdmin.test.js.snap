// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactAdmin route renders the complete page 1`] = `
<section
  className="ContactAdmin"
>
  <h2>
    Contact Admins
  </h2>
  <hr />
  <h4>
    Send us your suggestions, comments, or general inquiries! If you want to suggest a new trail to be included, please include as much details as you can including a name, address, and a website.
  </h4>
  <ContactForm />
</section>
`;
